### DISCUSSION TITLE
Can't create an feature request from element plus official site

### DISCUSSION THREAD

**Discussion by mockingjet** [element-plus/element-plus#8216] [General]
[My feature reqeust link](https://github.com/element-plus/element-plus/issues/new?body=%3C%21--+generated+by+https%3A%2F%2Fissue.element-plus.org%2F+DO+NOT+REMOVE+--%3E%0A%0AFeature+type%3A+**%60Enhancement%60**%0A%0A%23%23+Details%0A%0A-+Enhanced+compone+nt+name%3A+%60input-number%60%0A%0A%23%23+Description%0A%0AIt%27s+nice+to+offer+such+a+number+input+component%2C+thanks.%0A%0ABut+sometimes+we+just+want+users+to+only+use+controls+to+add+%2F+subtract+value.+Although+there+is+%60step-sctrictly%60+to+limit+user+input+value%2C+it+is+still+not+we+want.%0A%0AWe+want+an+option+to+disabled+the+middle+input+or+make+it+readonly.%0A%0A%23%23%23+Additional+comments%0A%0A_%28empty%29_%0A%0A%3C%21--+If+you+need+to+insert+a+screenshot%2C+please+paste+it+here.+--%3E&title=%5BFeature+Request%5D+%5BEnhancement%5D+%5Binput-number%5D+ElInputNumber+can+render+a+readonly+input%2C+but+still+can+change+value+by+control&labels=Community%3A%3AFeature+Request)

Above is the link generated from element-plus. Don't know why I can not create feature reqeust from that.

---
updates: it works now

**Comment by sxzz** [comment]
Maybe it's a bug of GitHub.
